maintenance/scripts/setup.sh

```diff
@@ -23,4 +23,4 @@ cp .env.example .env --preserve=all
 cd /var/www/html
 npm install
 npm run build
-chmod -R a+r,a+w node_modules apps/web/.turbo frontend/common/.turbo
+chmod -R a+r,a+w node_modules apps/web/.turbo
```

Comment:
Could you add the new .turbo directories to this step?
![image](https://user-images.githubusercontent.com/8978655/219163498-5953aa7c-8230-478c-b8f5-2a2777d4644a.png)
This worked for me.
![image](https://user-images.githubusercontent.com/8978655/219167817-d1d2045e-755a-40e4-8e85-96fbec75e4c2.png)

Reply:
oops, yup! Fixed in [update turbo permissions on setup](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5697/commits/8fc3b2d6795d00f5541cc4aef291c345ff367a39)